modules/ding_ipe_filter/ding_ipe_filter.install

```diff
@@ -140,3 +140,11 @@ function ding_ipe_filter_update_7106(&$sandbox) {
   variable_set('ding_ipe_filter_panes_selected', $new_selections);
 }
 
+/**
```

Comment:
Can't this be moved to easyddb_frontend_layout?

Reply:
https://github.com/easySuite/easyddb_frontend_layout/pull/22